actors/abi/sector.go

```diff
@@ -228,5 +228,5 @@ type WindowPoStVerifyInfo struct {
 
 // Information submitted by a miner to provide a Window PoSt.
 type OnChainWindowPoStVerifyInfo struct {
-	Proofs            []PoStProof
+	Proofs []PoStProof
```

Comment:
I know gofmt did this, but please try to avoid unrelated cosmetic changes. This just adds rebase pain.